scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -202,22 +250,22 @@ public static void manageUsers(String[] args) throws ManageUsersException {
     private static void askInteractively(Console console, UserInfo userInfo, Action action) {
         System.out.println(action);
         System.out.print("login: ");
-        if (userInfo.getLogin() == null) {
+        if (Strings.isNullOrEmpty(userInfo.getLogin())) {
```

Comment:
Creating a method named for example `isLoginSet()` to perform the tests can prevent the duplicated composition with _Strings.isNullOrEmpty_. It would also allow updating the condition in the future without looking at the whole code. Same apply to password.